README.md

```diff
@@ -11,7 +11,60 @@ The backend for a generic mobile-friendly quiz engine created using FastAPI and
 
 ### Local DB Setup
 
-- The following steps are for a Mac.
+#### Linux Systems
```

Comment:
Can you make a table of contents list at the top of readme file? I think we have enough sections / subsections. There are TOC generators online just use them instead of linking each section manually.